lib/fathead/java/parse_utils.py

```diff
@@ -69,6 +69,7 @@ def concat(clazz, description, url):
   ten = ''
   image = ''
   abstract = description.replace("\n", "\\n").replace("\t", "\\t") or "No abstract found"
+  abstract = '<section class="prog__container">' + abstract + '</section>'
```

Comment:
@lernae we should also wrap the text portion of the `abstract` in `<p>` -- that way if we also have any code blocks, it will be visually separated from them:

```html
<section class="prog__container">
    <p>description text</p>
    <pre><code>var codeBlock = awesome</code></pre>
</section>
```